nilearn/interfaces/fmriprep/load_confounds_utils.py

```diff
@@ -18,6 +19,7 @@
     "aroma": "_desc-smoothAROMAnonaggr_bold",
     "nii.gz": "(_space-.*)?_desc-preproc_bold.nii.gz",
     "dtseries.nii": "(_space-.*)?_bold.dtseries.nii",
+    "tedana": "_desc-optcom_bold.nii.gz",
```

Comment:
I noticed #5386 has comments expressing concerns for file naming stability. tedana has a file-name registry that lets users output files with (currently) 2 naming conventions. (Naming conventions are in : https://github.com/ME-ICA/tedana/blob/main/tedana/resources/config/outputs.json ).
You're using the BIDS naming convention here, which is fine if you assume everyone using nilearn with tedana will be using BIDS. If you want to make this more robust, `tedana` should output a file that ends `registry.json` and that file will contain the accurate file name value for the "combined img" key.

Reply:
Thanks for pointing this out! Currently we are only aiming to support BIDS-like conventions.